ci/release.hcl

```diff
@@ -0,0 +1,10 @@
+source = ["./build/darwin/dcos"]
+bundle_id = "com.mesosphere.dcos.cli"
+
+sign {
+  application_identity = "Developer ID Application: Christopher Molosso (LUBCYUGDL9)"
+}
+
+zip {
```

Comment:
For the main DC/OS CLI, wouldn't it make more sense to have a `.pkg`? It is normally a one-time installation (as the CLI is cluster independent) and this would provide the best user experience. cc @tillt

Reply:
you mean a "dmg" ?
Could be that we need to fix something on the CI system then. I bet there is a user shown "allow prompt" within the gon process of building the dmg

Reply:
No, @tillt might have more information but as far as I understood his explanation, DMG won't work for the CLI or won't provide the desired UX.

Reply:
Au contraire @sebbrandt87, a DMG or PKG allows stapling -- air gapped installations. Using a DMG is not advisable as it forces the user to copy the CLI and all of its plugins manually into the destination folder. When choosing between those two, the right choice for anything non UI related IN MY HUMBLE OPINION is a PKG installer -- that is what people commonly expect.

Reply:
I never expected pkg but zip

Reply:
I do not understand this last comment and I thus reiterate my question as it has not been answered yet by someone responsible for the DC/OS CLI deliverables: wouldn't it make more sense to have a `.pkg`? 

As Till added and tested, only `.dmg` and `.pkg` allow air-gapped installations and `.pkg` [is the format of choice to install software and other files onto a certain device](https://en.wikipedia.org/wiki/.pkg).

Even if the answer is "yes, a  `.pkg` would be better for the user experience" it might not fit with our delivery pipelines or our current documentation, both being fine reasons not doing it. I would just like an answer confirming that we make the creations of a `.zip` (replacing the non-signed binary from the past) instead of a `.pkg` a conscious choice.

Reply:
We picked the binary for the exact reason being that the documentation and the installation guide in the ui do "require" the binary. In the future I am happy to explore the DMG and pkg options as an alternative and then change the documentation.